gef.py

```diff
@@ -5645,7 +5645,19 @@ def search_pattern_by_address(self, pattern, start_address, end_address):
             else:
                 chunk_size = step
 
-            mem = read_memory(chunk_addr, chunk_size)
+            try:
+                mem = read_memory(chunk_addr, chunk_size)
+            except gdb.error as e:
+                estr = str(e)
+                if estr.startswith("Cannot access memory "):
+                    #
+                    # This is a special case.
+                    # See: https://github.com/hugsy/gef/issues/674
+                    #
+                    err(estr)
+                    return []
+                else:
```

Comment:
What other errors can we get?

Reply:
@Grazfather I don't yet know. But I think it is fine to throw away the other errors to the higher call stack if there is any.

Reply:
It is, I was just wondering if you had seen others.